.travis.yml

```diff
@@ -6,12 +6,50 @@ sudo: required
 services:
   - docker
 env:
-install: 
+  global:
+    - PA_TEST_ONLINE_INSTALLER=true
+    - PYTHONPATH=$(pwd)
+    - LONG_PRODUCT_TESTS="tests/product/test_server_install.py tests/product/test_status.py tests/product/test_collect.py tests/product/test_connectors.py tests/product/test_control.py tests/product/test_server_uninstall.py"
+  matrix:
+    - OTHER_TESTS=true
+    - SHORT_PRODUCT_TEST_GROUP=0
+    - LONG_PRODUCT_TEST_GROUP_STANDALONE_PRESTO_ADMIN="tests/product/test_server_install.py"
```

Comment:
I'd get rid of the word standalone in all the test group names.  It's the only way to run presto-admin right now, and makes the name really long.  
(Sorry for chiming in so late, I've been using this patch to be able to run tests on travis, and noticed this.  Also--works great!  thanks!)